### ISSUE TITLE
requesting pages twice

### ISSUE THREAD

**Issue by digininja** [sullo/nikto#559]
I''m running a scan through Burp and I just noticed that when you are brute forcing content you are making every request twice, is that how it is supposed to work?

Apart from the order, the requests for the pages are the same, does the ordering sometimes make a difference or is this a bug?

```
GET https://sub.site.com.au:443/sub.site.com.sql HTTP/1.1
User-Agent: Mozilla/5.00 (Nikto/2.1.6) (Evasions:None) (Test:sitefiles)
Host: sub.site.com.au
Connection: close
```

```
GET https://sub.site.com.au:443/sub.site.com.sql HTTP/1.1
User-Agent: Mozilla/5.00 (Nikto/2.1.6) (Evasions:None) (Test:sitefiles)
Connection: close
Host: sub.site.com.au
```

![nikto](https://user-images.githubusercontent.com/117081/43949878-b1c14ae4-9c86-11e8-98d5-bd3a18fd2112.png)

**Comment by digininja** [comment]
I've also just seen this roll through where again, the only difference was the order of the four request headers and the test number in the UA, the number ranged from 1427 to 1446. Unless something odd is being done with non-HTTP traffic it looks like a lot of wasted requests to me.

![nikto2](https://user-images.githubusercontent.com/117081/43950815-541047c6-9c89-11e8-92f8-6eae56d3c1ee.png)

**Comment by sullo** [comment]
So this plugin will request things with the Host header set and without it (just IP), so it does make (sort of) duplicate requests. I'll have to run it through a proxy to be sure (don't have Burp on this machine), but when I run with debug I see one with the hostname and one without.

**Comment by digininja** [comment]
I'll have a look and see.

**Comment by sullo** [comment]
So I did fix a few bugs in 044cd00bcc0acd0c15e59610566a7083964e4469 that I think were incidentally  causing this issue. I'm going to close this but reopen or open new if it is seen again. I'll try to keep an eye on it as well.
